tdrs-backend/tdpservice/search_indexes/admin/mixins.py

```diff
@@ -54,12 +54,17 @@ def export_as_csv(self, request, queryset):
         meta = self.model._meta
         field_names = [field.name for field in meta.fields]
 
+        if queryset.exists():
+            model = queryset.first()
```

Comment:
minor critique on this - since it formats the file name based on the first record in the queryset, it's technically possible to generate a file with records spanning many `quarter`s and `year`s, making the name a bit inaccurate. I think it's fine in general though

Reply:
agree @jtimpe; others may know this as well, but even more specifically: its based on the first record on the page. resorting the time period column on the same page, or going to another page that has a record for a different fiscal period is enough to change the filename. 

this can be revisited in a future ticket since the MVP for db access for data team is to query by fiscal period.